saleor/discount/migrations/0051_detach_sale_from_permission.py

```diff
@@ -0,0 +1,95 @@
+from django.contrib.contenttypes.management import create_contenttypes
+from django.db import DEFAULT_DB_ALIAS, migrations
+from django.apps import apps as global_apps
+from django.contrib.auth import get_permission_codename
+
+
+def handle_permissions(apps, _schema_editor):
+    app_config = apps.get_app_config("discount")
+    app_config.models_module = True
+    create_permissions(app_config, apps=apps)
+    app_config.models_module = None
+
+
+def create_permissions(
```

Comment:
Seems like it could cause a downtime. I'm wondering if it shouldn't be done in the celery task

Reply:
I don't think it is necessary, as it creates 82 records.